tests/testthat/dummy_packages/clean/R/clean_generics.R

```diff
@@ -25,3 +25,9 @@ drink_me.list <- function(x, ...) {
 drink_me.data.frame <- function(x, ...) {
   NULL
 }
+
+#' head on my_s3_object
+#' @export
+head.my_s3_object <- function(x, ...) {
+  NULL
+}
```

Comment:
This seems unrelated to the PR. Please remove it.

Reply:
Yup, sorry about that — I've removed it.

Reply:
Wouldn't utils always be loaded and thus `head` correctly identified as a generic regardless of the NAMESPACE because utils is a base package?

Reply:
it's possible to start R without utils IINM. e.g. Rscript doesn't load methods, I believe there's an option to control the other default packages as well

Reply:
that's why R CMD check warns about not putting head (and other functions from default packages) in the NAMESPACE IIRC